src/App.js

```diff
@@ -1,8 +1,76 @@
 import React from 'react'
 
 const App = () => (
-  <div>
-    <h1>REACT AND WEBPACK LIVE!!!</h1>
+  <div className='container'>
```

Comment:
Can you break this up on smaller components?

Reply:
yes that is my next step, I just wanted to get it piped over from the old project correctly first.